phenopacketlab-frontend/src/environments/environment.ts

```diff
@@ -2,10 +2,12 @@
 // `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
 // The list of file replacements can be found in `angular.json`.
 
+const URL = 'http://localhost:8080';
 const API_URL = 'http://localhost:8080/api/v1';
```

Comment:
API_URL = URL+'/api/v1';

Reply:
good point

Reply:
Modified in [ef91e0](https://github.com/TheJacksonLaboratory/PhenopacketLab/pull/179/commits/ef91e0029799570f98014495acd15196609abe91)